app/controllers/admin/environments_controller.rb

```diff
@@ -4,11 +4,9 @@ class Admin::EnvironmentsController < ApplicationController
   before_action :environment, only: [:edit, :update, :destroy]
 
   def index
-    @environments = Environment.all
-
     respond_to do |format|
       format.html
-      format.json { render json: @environments }
+      format.json { render json: Environment.all }
```

Comment:
then I'd have to include it in the controller :(

On Wed, Apr 20, 2016 at 3:07 PM, Pierre Schambacher <
notifications@github.com> wrote:

> In app/controllers/admin/environments_controller.rb
> <https://github.com/zendesk/samson/pull/858#discussion_r60496333>:
>
> >      respond_to do |format|
> >        format.html
> > -      format.json { render json: @environments }
> > +      format.json { render json: Environment.all }
>
> why not use the application helper then?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/858/files/2dbcfd28a5602738743e5661898c221d37da5da9#r60496333>
>